documentation/reference/feature-flags.md

```diff
@@ -0,0 +1,21 @@
+# Feature Flags Supported by Charmed Superset
+
+There are a number of features that can be enabled/disabled via the Superset charm:
+```
```

Comment:
Add this above the command and below the table:
"You can enable, `True`, and/or disable, `False`, these flags using the following command:"